### DISCUSSION TITLE
Benchmark: MTP, NEP, and NNMTP on a Ti-V-Zr-Nb high-entropy alloy

### DISCUSSION THREAD

**Discussion by lhycms** [lhycms/AI2Pot#22] [General]
We benchmarked **MTP, NEP, and NNMTP** on a Ti-V-Zr-Nb high-entropy alloy (HEA) system.

The training dataset was taken from the following work:

> [https://doi.org/10.1016/j.matdes.2026.116561](https://doi.org/10.1016/j.matdes.2026.116561)

## 1. Model settings

The key parameters used for each model are listed below.

### 1.1 MTP

```txt
mtp_level = 18
rmax = 5.5
zbl_rmax = 2.0
rmin = 1.0
```

### 1.2 NEP

```txt
n_radial_basis = 8
n_angular_basis = 6
num_neurons = 50
rmax_radial = 6.0
rmax_angular = 4.0
zbl_rmax = 2.0
```

### 1.3 NNMTP

```txt
mtp_level = 18
rmax = 5.5
num_neurons = 30
zbl_rmax = 2.0
rmin = 0.0
```

Each model was trained using two different batch sizes:

- `batch_size = 1`
- `batch_size = 16`

### 2.1 MTP

<img width="1132" height="318" alt="MTP batch_size=1" src="https://github.com/user-attachments/assets/73329a6e-5323-4eef-8637-b9944b3d6790" />

### 2.2 NEP

<img width="1134" height="304" alt="NEP batch_size=1" src="https://github.com/user-attachments/assets/7de7adc9-24b4-4e1c-a51b-396ae151a301" />

### 2.3 NNMTP

<img width="1134" height="320" alt="NNMTP batch_size=1" src="https://github.com/user-attachments/assets/a6aa873f-599c-4902-aa09-22e817436a8e" />

### 3.1 MTP

<img width="1134" height="316" alt="MTP batch_size=16" src="https://github.com/user-attachments/assets/d92ecd8f-b3e7-4b77-8038-c2df6c86241e" />

### 3.2 NEP

<img width="1136" height="321" alt="NEP batch_size=16" src="https://github.com/user-attachments/assets/19496190-8b2e-4138-93b7-24412b616764" />

### 3.3 NNMTP

<img width="1136" height="321" alt="NNMTP batch_size=16" src="https://github.com/user-attachments/assets/1b6875d8-85ff-407b-875d-c4bc6421f57f" />

## 4. Summary

These results provide a direct comparison of the fitting behavior of MTP, NEP, and NNMTP on the same multicomponent HEA dataset, as well as the influence of `batch_size` on the final fitting accuracy.
